tools/release-init.sh

```diff
@@ -18,9 +18,9 @@ git push -u origin release/${realm_version}
 git checkout -b prepare-$realm_version
 
 # update dependencies.list
-sed -i "s/^VERSION.*/VERSION=${realm_version}/" ${project_dir}/dependencies.list
+sed -i -e "s/^VERSION.*/VERSION=${realm_version}/" ${project_dir}/dependencies.list
```

Comment:
You're right! I didn't catch that `dependencies.list-e` was being created on Mac. I've updated it according to [this advice](You're right! I didn't catch that `dependencies.list-e` was being created on Mac. I've updated it according to https://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux) and then remove the backups as suggested. 

I also linted with `shellcheck`and fixed a few places where paths weren't being quoted which could result in word splitting if someone had spaces in their paths.